src/content/docs/dev/send-tokens/interchain-tokens/integrate-custom-token.mdx

```diff
@@ -98,13 +104,26 @@ function linkToken(
 ) external payable returns (bytes32 tokenId) {}
 ```
 
-This function links a remote token on `destinationChain` to a local token corresponding to the `tokenId` computed from the provided `salt`.
+This function does several important things:
+
+1. It computes a `tokenId` using the same salt you used during token registration
+2. Creates a cross-chain message that instructs the destination chain to deploy a Token Manager for your token
+3. Establishes the connection between your tokens across chains by associating them with the same `tokenId`
+4. Sets up the appropriate [Token Manager](/dev/send-tokens/interchain-tokens/token-manager/) type that defines how tokens will be bridged (lock/unlock, mint/burn, etc.)
+
+When this function executes, it:
```

Comment:
we should be consistent. here we list with dashes but previously we list with numbers